tools/RAiDER/models/weatherModel.py

```diff
@@ -847,34 +846,18 @@ def write(
 
 
 def make_weather_model_filename(name, time, ll_bounds):
-    if ll_bounds[0] < 0:
-        S = 'S'
-    else:
-        S = 'N'
-    if ll_bounds[1] < 0:
-        N = 'S'
-    else:
-        N = 'N'
-    if ll_bounds[2] < 0:
-        W = 'W'
-    else:
-        W = 'E'
-    if ll_bounds[3] < 0:
-        E = 'W'
-    else:
-        E = 'E'
-    return '{}_{}_{:.0f}{}_{:.0f}{}_{:.0f}{}_{:.0f}{}.nc'.format(
-        name,
-        time.strftime("%Y_%m_%d_T%H_%M_%S"),
-        np.ceil(np.abs(ll_bounds[0])),
-        S,
-        np.ceil(np.abs(ll_bounds[1])),
-        N,
-        np.ceil(np.abs(ll_bounds[2])),
-        W,
-        np.ceil(np.abs(ll_bounds[3])),
-        E
-    )
+    s = np.floor(ll_bounds[0])
+    S = f'{np.abs(s):.0f}S' if s <0 else f'{s:.0f}N'
+
+    n = np.ceil(ll_bounds[1])
+    N = f'{np.abs(n):.0f}S' if n <0 else f'{n:.0f}N'
+
+    w = np.floor(ll_bounds[2])
+    W = f'{np.abs(w):.0f}W' if w <0 else f'{w:.0f}E'
+
+    e = np.ceil(ll_bounds[3])
+    E = f'{np.abs(e):.0f}W' if e <0 else f'{e:.0f}E'
+    return f'{name}_{time.strftime("%Y_%m_%d_T%H_%M_%S")}_{S}_{N}_{W}_{E}.nc'
```

Comment:
What are you doing here? As in I thought your work only was generating a test?

Reply:
@jlmaurer Does this break things/ backward compatible?

Reply:
@bbuzz31 I would tend to agree here in terms of not changing this since this PR needs to get through as quickly as possible, but if you have sufficient tests then its probably fine.